hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -172,6 +183,14 @@ private void scan() {
             log(backupTimeouts, invocationTimeouts);
         }
 
+        private boolean isCallIdsMatch(Long callId, Invocation invocation) {
```

Comment:
Not very important but I would call this method callIdMatches. And why don't you use a primitive long?